.github/workflows/javascript.yml

```diff
@@ -0,0 +1,94 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2023 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: JavaScript Pipeline
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  lint:
+    name: Perform linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "lts/iron"
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: npm-${{ hashFiles('package-lock.json') }}
```

Comment:
Last question: I guess we need the path `benchexec/tablegenerator/react-table/package-lock.json` here?

Reply:
Yes totally. I'm so sorry for missing it and not being careful!

Reply:
No problem! Now it works perfectly:
![image](https://github.com/sosy-lab/benchexec/assets/2545335/874a02bd-a097-4eaa-b061-c52d9bc92d7b)
It shows that the primary key has the hash (was missing previously), but that it successfully used the cache named `npm-`.